packages/ethereum/README.md

```diff
@@ -0,0 +1,43 @@
+# Ethereum AuthMethod and Verifier
+
+
+## Installation
+
+```
+npm install --save ethereum-cacao
+```
+
+## Usage
+
+```ts
+// Auth Usage
+import { EthereumWebAuth, getAccountId } from 'ethereum-cacao'
```

Comment:
```suggestion
import { EthereumWebAuth, getAccountId } from 'did-pkh-eip155'
```

Rather than calling these packages something related to CACAO I think using DID PKH is more appropriate.

Reply:
yeah kinda torn on this, i think did pkh wording is more useful to relay its purpose to developers, but technically its very specific to cacao, you could have more general did pkh signers/verifier after

Reply:
Imo, catering to devs should be top prio.